discovery/db_discovery.go

```diff
@@ -137,6 +138,10 @@ func (dbo *DBOrchestratorPoolCache) cacheTranscoderPool() error {
 	}
 
 	for _, o := range orchestrators {
+		maxInt64 := big.NewInt(int64(math.MaxInt64))
```

Comment:
What do you think about moving this logic into `ethOrchToDBOrch()` and creating an instance of `DBOrch()` with the correct value for `DeactivationRound` instead of mutating each instance of `lpTypes.Transcoder` in this loop? Then, we could also add a unit test for `ethOrchToDBOrch()` that confirms that we are correctly setting `DeactivationRound` on `DBOrch` to max int64 whenever `DeactivationRound` on `lpTypes.Transcoder` is equal to max uint256.

Reply:
fixed in 44a7cf9 

also using a const for `maxInt64` (as  `int64`) so that it is evaluated at compile time rather than runtime.